app/src/helpers/PaperTraderApi.ts

```diff
@@ -79,8 +75,8 @@ export default class PaperTraderApi {
   /**
    * Makes a request to server with stock ticker to get a quote
    */
-  static async getStock(ticker: string): Promise<FinnhubQuote> {
-    const res = (await this.request(`stock/search?ticker=${ticker}`)).quote;
+  static async getStock(ticker: string): Promise<number> {
```

Comment:
```suggestion
  static async getStockPrice(ticker: string): Promise<number> {
```

might be more clear since `getStock` makes me think you might be returning a `Stock` object with additional fields (which might not be a bad idea to make it a class with only a single field for now 🤔 )

Reply:
If I were to make a `Stock` class, how would it be used?

Also, would the class be something like this? 
```
class Stock {
    constructor(price) {
        this.price = price
}
```

Reply:
exactly. It won't make life any easier now, but it will be much simpler to add additional fields if you decide to expand it in the future (eg. `stock.companyName, stock.ticker`